CWE-94-Python_darwin/pedrohbtp_pytorch-zappa-serverless/main.py

```
@@ -0,0 +1,130 @@
+
+def pickle_obj(data, path):
+    with open(path,'wb+') as f:
+        pickle.dump(data,f)
+
+def unpickle_obj(path):
+    with open(path,'rb') as f:
+        data = pickle.load(f)
+    return data
+
+def unpickle_s3_obj(bucket, path):
+    if IS_LOCAL == False:
```

Ruff (0.13.2)

Avoid equality comparisons to `False`; use `not IS_LOCAL:` for false checks

Replace with `not IS_LOCAL`

(E712)